charts/osm/crds/meshconfig.yaml

```diff
@@ -227,3 +227,6 @@ spec:
                     enableValidatingWebhook:
                       type: boolean
                       default: false
+                    enableCrdConverter:
+                      type: boolean
+                      default: false
```

Comment:
Can we drop this from the CRD because it is only used in the YAML files and not Go code?

Reply:
makes sense, I have removed it